analysis/topeft_run2/datacards_post_processing.py

```diff
@@ -96,19 +114,18 @@ def main():
         for line in lines_from_condor_out_to_print:
             print(f"\t\t* In {line[0]}: {line[1]}")
 
-    ####### Copy the TOP-22-006 relevant files to their own dir ######
-
 
-    with open(topeft_path("channels/ch_lst_test.json"), "r") as ch_json:
+    ####### Copy the TOP-22-006 relevant files to their own dir ######
+    with open(topeft_path("channels/ch_lst.json"), "r") as ch_json:
         select_ch_lst = json.load(ch_json)
         #reading the macro analysis setup
         if args.set_up_top22006:
             import_sr_ch_lst = select_ch_lst["TOP22_006_CH_LST_SR"]
-        if args.set_up_offZdivision:
+        elif args.set_up_offZdivision:
```

Comment:
Since we only allow one flag at a time, is the `elif` really necessary?

Reply:
I think it's useful to avoid running an unneeded `if` statement